qa/tasks/cephfs/test_volumes.py

```diff
@@ -2692,6 +2692,25 @@ def test_subvolume_snapshot_clone(self):
         # verify trash dir is clean
         self._wait_for_trash_empty()
 
+    def test_subvolume_snapshot_reconf_max_concurrent_clones(self):
```

Comment:
````
2021-01-05T20:18:17.043 INFO:teuthology.orchestra.run.smithi135.stderr:2021-01-05 20:18:17.042 7f0427fff700  1 -- 172.21.15.135:0/3696700857 <== mon.0 v2:172.21.15.110:3300/0 7 ==== mon_command_ack([{"prefix": "config get", "who": "mgr", "key": "mgr/volumes/max_concurrent_clones"}]=-22 unrecognized entity 'mgr' v160) v1 ==== 141+0+0 (secure 0 0 0) 0x7f0430000bf0 con 0x7f0440098e40
2021-01-05T20:18:17.045 INFO:teuthology.orchestra.run.smithi135.stderr:Error EINVAL: unrecognized entity 'mgr'
````

From: /ceph/teuthology-archive/yuriw-2021-01-05_18:17:46-fs-wip-yuri-testing-2021-01-05-0841-nautilus-distro-basic-smithi/5757402/teuthology.log

I think in nautilus you need to give a particular mgr id. I think it should be safe to use `mgr.x` for the purposes of getting this nautilus backport to work.

Reply:
Done